_typos.toml

```diff
@@ -0,0 +1,9 @@
+[default.extend-words]
+monitord = "monitord"
+
+# All exceptions below should be removed when upstream will be fixed
+includeSytem = "includeSytem"
+Sytem = "Sytem"
+cssPsuedoElementExclusion = "cssPsuedoElementExclusion"
+Psuedo = "Psuedo"
+criterias = "criterias"
```

Comment:
should be criterion/criteria instead of criteria/criterias, but we can skip it for now

Reply:
as we're not able to test the glpi plugin